buildAddons.cmd

```diff
@@ -2,6 +2,7 @@ call :pauseOnError msbuild /p:configuration=Release /t:clean
 call :pauseOnError msbuild /p:configuration=Release /t:BuildAllConfigurations
 call :pauseOnError msbuild /p:configuration=Release /t:BuildAKVNetFx
 call :pauseOnError msbuild /p:configuration=Release /t:BuildAKVNetCoreAllOS
+call :pauseOnError msbuild /p:configuration=Release /t:BuildAKVNetStandardAllOS
```

Comment:
I find `BuildAKVNetStAllOS` in build.proj . Can we keep the consistency?

Reply:
My typo